catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/AssociationsAttributes.java

```diff
@@ -26,23 +27,25 @@
  * This class provides attributes that represent associations between products.
  */
 public class AssociationsAttributes implements Associations, MetacardType {
-    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+    private static final Set<AttributeDescriptor> DESCRIPTORS;
 
     private static final String NAME = "associations";
 
     static {
-        DESCRIPTORS.add(new AttributeDescriptorImpl(DERIVED,
+        HashSet<AttributeDescriptor> descriptors = new HashSet<>();
```

Comment:
Is there a specific reason you are declaring descriptors to be a HashSet and not a set?  Why not:
` Set<AttributeDescriptor> descriptors = new HashSet<>();`?